sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -800,6 +802,7 @@ public void recognizePiiEntitiesDiacriticsNfd(HttpClient httpClient, TextAnalyti
 
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("com.azure.ai.textanalytics.TestUtils#getTestParameters")
+    @Disabled("아가 means baby but recognized as an person. Previously, this is not recognized as Pii category. https://github.com/Azure/azure-sdk-for-java/issues/22208")
```

Comment:
Having these errors means you need to update your tests? or is there any regression or bug in the service?

Reply:
It seems more on the service side regression. Will create a service issue.